lib/flippant/ruleset.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Flippant.Ruleset do
+  @moduledoc """
+  Rulesets determine whether the actor is a member of a group, and enabled for a given rule.
+  """
+
+  alias Flippant.Rules
+
+  @doc """
+  Validate that an actor is both a member of `group` and in the `enabled_for` list. For example,
+  if we had a `staff` group, and a rule containing `%{"staff" => ids}`, we could have a function
+  like this:
+
+  ```
+    def actor_in_group?("staff", enabled_for, %{staff: true} = actor) do
+      Enum.any?(enabled_for, &actor.id == &1.id)
+    end
+  ```
+
+  This both validates the actor is staff, and that the actor is in the enabled list. Allowing
+  ruleset definition in this way makes the logic for rule checking very visible in application
+  code.
+  """
+  @callback actor_in_group?(group :: binary(), enabled_for :: list(), actor :: Rules.actor()) ::
```

Comment:
This is the primary interface for users of the library now. I think we can use a clearer name, like `enabled?`. What do you think?

Reply:
Sounds good to me